columnRow.js

```diff
@@ -0,0 +1,67 @@
+const WIDTH_SETTINGS = {
+  minSizeValueCol: 39,
+  minSizeValuePixel: 370,
+  normalSizeValue: 10,
+  extraSizeValue: 5,
+  extraUnitLocation: [4]
+};
+
+const HEIGHT_SETTINGS = {
+  minSizeValueRow: 12,
+  minSizeValuePixel: 300,
+  normalSizeValue: 20,
+  extraSizeValue: 25,
+  extraUnitLocation: [2, 7]
+};
```

Comment:
`minSizeValueCol/Row`: minimum value for input. The logic to convert only works if starting here.
`minSizeValuePixel`: equivalent in pixels for `minSizeValueCol/Row`. _Hyper don't accept a smaller number_.
`normalSizeValue`: value of each columns/rows in pixels.
`extraSizeValue`: value of each `extraUnitLocation`.
`extraUnitLocation`: unit with different equation logic. Example:

extra width: ignore all columns ending with `extraUnitLocation`. 
(if user input contains `extraUnitLocation` as last number, width gets `extraSizeValue` once)

extra height: all rows ending with `extraUnitLocation` get `extraSizeValue` instead of `normalSizeValue`.